<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-U1JVRC1" failures="1" tests="2" name="Default test(failed)" time="0.1" errors="0" timestamp="2020-10-14T19:59:47 IST">
  <testcase classname="PriorityDemo.TestNGRetryFailuresDemo" name="test2" time="0.016"/>
  <testcase classname="PriorityDemo.TestNGRetryFailuresDemo" name="test3" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at PriorityDemo.TestNGRetryFailuresDemo.test3(TestNGRetryFailuresDemo.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="PriorityDemo.TestNGRetryFailuresDemo" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
